### PR TITLE
Add sum to timer, add logging to debug missing histogram

### PR DESCRIPTION
**Pull request by burmanm**

### CODE REVIEW

File: management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java [hunk 1]
```diff
@@ -269,12 +269,14 @@ private void setTimerFiller(Timer timer, CassandraMetricDefinition proto, Cassan
 
                     buckets = (long[]) decayingHistogramOffsetMethod.invoke(snapshot);
                 } catch (InvocationTargetException | IllegalAccessException | NoSuchMethodException e) {
-                    throw new RuntimeException(e);
+                    logger.error(String.format("Unable to getOffsets for DSE, snapshotClass: %s", snapshotClass), e);
+//                    throw new RuntimeException(e);
```
Comment (adejanovski) [inline management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java:273]: Should this be thrown?
If not, could you remove that line?

**Review by adejanovski — APPROVED** [review]
Just one comment to remove (or uncomment), otherwise looks good 👍

File: management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java (hunk 1 above)
Comment (burmanm) [reply management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java:273]: I'm not sure if we gain anything by throwing it. Even logging should perhaps be delegated to debug level.

Comment (adejanovski) [reply management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java]: If users cannot act on the problem and this is not a blocking error, then it should move at debug level indeed 👍 
You can remove the commented line if there's no need to re-throw the error.

**Pull request merged**